src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -118,6 +118,30 @@ class TimeSeriesRDD(val index: DateTimeIndex, parent: RDD[(String, Vector[Double
   }
 
   /**
+   * Takes the first N series in the TimeSeriesRDD and returns a new, reduced, TimeSeriesRDD.
+   * Works by creating an array of series to include and then checks for membership.
+   * @param n the number of series to take (first n taken) n >= 0
+   * @return reduced TimeSeriesRDD
+   */
+  def takeNSeries(n: Int): TimeSeriesRDD = {
```

Comment:
Yeah, that was one issue I had with the naming. Do you think it is a useful function at all? I was just envisioning someone how might want to just keep the first 1000 or whatever series but didn't want to collect.

I was using it in the portfolio optim example, but it is small enough I can just take it out and use as a separate function.